### DISCUSSION TITLE
Is there a way to use another solver?

### DISCUSSION THREAD

**Discussion by erelsgl** [coin-or/python-mip#358] [Q&A]
I would like to use mip with MOSEK or XPRESS, instead of CBC or GUROBI. Is it possible?

**Comment by rschwarz** [comment]
For "normal" use of the `mip` library, each solver needs their own implementation of the `Solver` interface.

As a work-around (for experiments), one option would be to create the model with Cbc, then write the problem to a file (`.lp` or `.mps`) and load that file with another solver.
